### PR TITLE
Ch 5 conditionals

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/decisions-with-conditionals/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
```
Comment (linwang314) [inline content/decisions-with-conditionals/_index.md:16]: Add details to Learning Objectives and Key Terminology

File: content/decisions-with-conditionals/studio/_index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # Wed Jul 5 08:49:19 AM CDT 2023
+---
+
+In this studio, you are going to write code to verify that your Space Shuttle is prepared for LiftOff! This activity will combine knowledge you have gained from this chapter and the [Data and Variables Chapter]({{< relref "../../data-and-variables/_index.md" >}})
+
+{{% notice blue Note "rocket" %}}
+The starter code for this studio can be found inside of the following repository: [javascript-projects](https://github.com/LaunchCodeEducation/javascript-projects/) within the `booleans-and-conditionals` directory.
```
Comment (linwang314) [inline content/decisions-with-conditionals/studio/_index.md:18]: The javascript-projects repository doesn't yet have a booleans-and-conditional directory.

File: content/decisions-with-conditionals/_index.md (hunk 1 above)
Comment (jwoolbright23) [reply content/decisions-with-conditionals/_index.md:22]: added learning objectives and key terms

File: content/decisions-with-conditionals/studio/_index.md (hunk 2 above)
Comment (jwoolbright23) [reply content/decisions-with-conditionals/studio/_index.md:18]: content exists it just has not been merged into the main branch quite yet.

**Pull request merged**